ds3-autogen-go/src/main/kotlin/com/spectralogic/ds3autogen/go/generators/client/ClientGeneratorUtil.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2017 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3autogen.go.generators.client
+
+import com.spectralogic.ds3autogen.api.models.apispec.Ds3Request
+import com.spectralogic.ds3autogen.api.models.enums.Classification
+import com.spectralogic.ds3autogen.api.models.enums.Requirement
+import com.spectralogic.ds3autogen.go.utils.getGoArgFromResource
+import com.spectralogic.ds3autogen.go.utils.isGoResourceAnArg
+import com.spectralogic.ds3autogen.utils.Ds3RequestClassificationUtil
+import com.spectralogic.ds3autogen.utils.Ds3RequestUtils
+import com.spectralogic.ds3autogen.utils.RequestConverterUtil
+import com.spectralogic.ds3autogen.utils.models.NotificationType
+
+private const val PATH_REQUEST_REFERENCE = "request"
+
+/**
+ * Creates the Go request path code for a Ds3 request
+ */
+fun toRequestPath(ds3Request: Ds3Request): String {
```

Comment:
These can be turned into extension functions, which will change the call sites of these methods to:

```kotlin

val ds3Request = Ds3Request()

val path = ds3Request.toRequestPath()
```

Reply:
Consider making this change for all three of these methods in this file